app/uploaders/post_image_uploader.rb

```diff
@@ -0,0 +1,21 @@
+##
+# The file uploader class for uploading images to an SSE website post
+class PostImageUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+
+  storage :file
+
+  ##
+  # Limits only images to be uploaded to an SSE website post
+  def extension_whitelist
+    ['jpg', 'jpeg', 'gif', 'png']
+  end
+
+  version :preview do 
+    process resize_to_limit: [800, 800]
```

Comment:
Can we replace these with static constants? It might be good to have some self-documenting descriptions to explain what these numbers stand for, and to easily find them for adjusting if necessary in the future.